flink-clients/src/main/java/org/apache/flink/client/deployment/ClusterClientJobClientAdapter.java

```diff
@@ -48,25 +54,45 @@
 
```

Comment:
This is not part of your PR, but in this class the `Logger` is not longer used so you can remove it,

Reply:
Make sense. Done.